extensions/amp-accordion/1.0/animations.js

```diff
@@ -111,8 +112,7 @@ export function animateCollapse(content) {
       );
     },
     () => {
-      // Undo the `content.hidden = true` above.
-      content.hidden = true;
+      content.classList.remove('i-amphtml-animating');
```

Comment:
Looks like `add` and `remove` for this class should go into the common `animate()` function below since it's always the same. But I could be wrong and something might not match up. But please try.

Reply:
Updated in latest revision.